src/stylesheets/application.scss

```diff
@@ -26,6 +27,8 @@
 @import 'components/wheel-select';
 @import 'components/rating';
 @import 'components/np-score';
+@import 'components/survey-input-container';
+@import 'components/select_search';
```

Comment:
@rafayet-monon Should we use `-` instead of `_`?

Reply:
Updated!